.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: ruby
+script: bin/rspec
+rvm:
+  - 1.9.3
```

Comment:
1.9.3 is now [EOL](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/).